src/libs/geolocation/hooks/useDistance.test.ts

```diff
@@ -1,20 +1,18 @@
-/* eslint-disable local-rules/independent-mocks */
-import { mocked } from 'ts-jest/utils'
-
 import { IGeolocationContext, useGeolocation } from 'libs/geolocation'
 import { useDistance } from 'libs/geolocation/hooks/useDistance'
 import { formatDistance } from 'libs/parsers'
 
 jest.mock('libs/parsers')
 
 jest.mock('libs/geolocation')
-const mockUseGeolocation = mocked(useGeolocation)
+const mockUseGeolocation = jest.mocked(useGeolocation)
 
 const position = { latitude: 90, longitude: 90 }
 const offerPosition = { lat: 31, long: 56 }
 
 describe('useDistance()', () => {
   it('should call useGeolocation and formatDistance when geolocation is on', () => {
+    // eslint-disable-next-line local-rules/independent-mocks
     mockUseGeolocation.mockReturnValue({ position } as IGeolocationContext)
```

Comment:
```suggestion
	const toto: IGeolocationContext = { position }
    // eslint-disable-next-line local-rules/independent-mocks
    mockUseGeolocation.mockReturnValue(toto)
```

je recommande de typer explicitement plutot que de faire un `as`